package.json

```diff
@@ -30,23 +30,23 @@
   },
   "homepage": "https://github.com/3fs/eslint-config#readme",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "4.12.0",
-    "@typescript-eslint/parser": "4.12.0",
+    "@typescript-eslint/eslint-plugin": "5.13.0",
+    "@typescript-eslint/parser": "5.13.0",
     "babel-eslint": "10.1.0",
-    "eslint": "7.17.0",
+    "eslint": "8.10.0",
     "eslint-find-rules": "4.1.0",
-    "eslint-plugin-angular": "4.0.1",
-    "eslint-plugin-jsx-a11y": "6.4.1",
-    "eslint-plugin-react": "7.22.0",
-    "eslint-plugin-react-hooks": "4.2.0",
-    "eslint-plugin-react-native": "3.10.0",
-    "typescript": "4.1.3"
+    "eslint-plugin-angular": "4.1.0",
+    "eslint-plugin-jsx-a11y": "6.5.1",
+    "eslint-plugin-react": "7.29.2",
+    "eslint-plugin-react-hooks": "4.3.0",
+    "eslint-plugin-react-native": "4.0.0",
+    "typescript": "4.6.2"
   },
   "peerDependencies": {
     "@angular-eslint/template-parser": ">=2.0.2 <13.0.0",
```

Comment:
We use 13.1.0 in our project, so this condition is making us some problems. Is it possible to change it? 🤔

Reply:
🤔 So we have 2 issues 😄 

Would it work if we open a second MR to fix this?

Reply:
Yes we can update this in separate MR. 👍